### ISSUE TITLE
Errors when generating non-square size

### ISSUE THREAD

**Issue by jasd3199** [Tencent/HunyuanDiT#100]
When I change the desired size to any different to width=height, the generation fails.

This is an example (1280x720):
![hunyuan_00034_](https://github.com/Tencent/HunyuanDiT/assets/80776713/612d9782-15ee-4653-9d7e-dd8451cb151f)

**Comment by zml-ai** [comment]
Hi, could you provide the inference script and model type? 

More information would help us quickly pinpoint the issue.

**Comment by jasd3199** [comment]
Yes, I'm using ComfyUI for the inference with one of the example workflows that the repo provides ([hunyuan_ksampler_api.json](https://github.com/Tencent/HunyuanDiT/blob/main/comfyui-hydit/workflow/hunyuan_ksampler_api.json)), only changing the desired size and prompts.

The model that I loaded was in: 
https://huggingface.co/Tencent-Hunyuan/HunyuanDiT/blob/main/t2i/model/pytorch_model_ema.pt

I had this problem using the initial version and the last version V1.1

**Comment by xuhuaren** [comment]
we will fix it and release code around 6.30.
